src/wx/cmdevents.cpp

```diff
@@ -2780,8 +2799,10 @@ EVT_HANDLER_MASK(ChangeIFB, "Change Interframe Blending", CMDEN_NREC_ANY)
         panel->panel = nullptr;
     }
 
+    auto ifb_name = ifb_short_name(interframe_blenders, gopts.ifb);
+
     wxString msg;
-    msg.Printf(_("Using interframe blending #%d"), gopts.ifb);
+    msg.Printf(_("Using interframe blending:") + " %s", ifb_name);
     systemScreenMessage(msg);
```

Comment:
You can use `wxString::Format` directly rather than using a variable.

Reply:
Also, put the `%s` formatter in the localized string too. As above, some languages don't follow the same word order.